src/types/IstioObjects.ts

```diff
@@ -722,6 +854,7 @@ export enum PeerAuthenticationMutualTLSMode {
   STRICT = 'STRICT'
 }
 
+// 1.6
 export interface WorkloadEntry extends IstioObject {
   address: string;
```

Comment:
This is nitpicking right now since we are not using WorkloadEntry objects into the UI. So, not a blocker itself.

I think this should be something like:

```
export interface WorkloadEntry extends IstioObject {
  spec: WorkloadEntrySpec
}
```

and then, in ServiceEntrySpec:
```
export interface ServiceEntrySpec {
   endpoints?: WorkloadEntrySpec[];
}
```

this way we don't include kind, metadata and apiVersion.

Reply:
It's not nitpicking, it's a good catch.
It was missed the "spec" field which could lead in a problem.
I'll fix it.

Reply:
And it also happens to other objects I see.
Thatnks, you spotted several issues, that's good.

Reply:
It happens same problem in PeerAuthentication, WorkloadEntry and RequestAuthentication.
I'll fix those 3 mappings as par of this PR.